### PR TITLE
Greek translation

### PR DESCRIPTION
**Pull request by makishart**
Hello I would like to contribute with a Greek translation

### CODE REVIEW

File: packages/translations/lib/l10n/app_gr.arb [hunk 1]
```diff
@@ -0,0 +1,327 @@
+{
+  "@@locale": "gr",
```
Comment (ferraridamiano) [inline packages/translations/lib/l10n/app_gr.arb:2]: Same here

File: lib/models/settings.dart [hunk 2]
```diff
@@ -9,6 +9,7 @@ final Map<Locale, String> mapLocale = {
   const Locale('de'): 'Deutsch',
   const Locale('es'): 'Español',
   const Locale('fr'): 'Français',
+  const Locale('gr'): 'Ελληνικά',
```
Comment (ferraridamiano) [inline lib/models/settings.dart:12]: I think that the CI failed because the language code for Greek is el and not gr

**Review by ferraridamiano — CHANGES_REQUESTED** [review]

File: lib/models/settings.dart (hunk 2 above)
Comment (makishart) [reply lib/models/settings.dart:12]: Yes, sorry for that

Comment (makishart) [reply lib/models/settings.dart:12]: Shall I make a new pull request?

Comment (makishart) [reply lib/models/settings.dart]: Ok, I changed it at el

**Comment by ferraridamiano** [comment]
Thank you!

**Pull request merged**